scripts/twitch-proxy/updateApiKey.js

```diff
@@ -14,7 +14,8 @@ const updateApiKey = async () => {
     .catch(err => console.log(err));
 
   // Write new file with updated token
-  fs.writeFileSync(envFilePath, `TWITCH_CLIENT_ID=${TWITCH_CLIENT_ID}
+  fs.writeFileSync(envFilePath, `PORT=${PORT}
```

Comment:
GitHub won't let me reach it, but doesn't `PORT` need to be destructured out of the `process.env` in line 6?

Seems like this would be undefined, currently.

Reply:
Doh, that's right :facepalm: Thanks for catching that @nhcarrigan.

I'll push up a fix for that now.